common/src/main/java/com/wynntils/services/mapdata/attributes/type/MapAttributes.java

```diff
@@ -6,32 +6,36 @@
 
 import com.wynntils.utils.colors.CustomColor;
 import com.wynntils.utils.render.type.TextShadow;
+import java.util.Optional;
 
 public interface MapAttributes {
     // If this is the empty string (""), then no label will be displayed
-    // null means inherit
-    String getLabel();
+    // empty means inherit
+    Optional<String> getLabel();
 
     // If this is MapFeatureIcon.NO_ICON_ID ("none"), then no icon will be displayed
-    // null means inherit
-    String getIconId();
+    // empty means inherit
+    Optional<String> getIconId();
 
-    // 1-1000, 1000 is highest priority (drawn on top of everything else)
-    // 0 means no value specified; inherit
-    int getPriority();
+    // 1-1000, 1000 is the highest priority (drawn on top of everything else)
+    // empty means no value specified; inherit
+    Optional<Integer> getPriority();
 
-    // 0 means no information is available, or suitable for all combat levels
-    int getLevel();
+    // the minimum combat level for which this feature is suitable for
+    // 0 means suitable for all levels
+    // -1 means no information is available
```

Comment:
```suggestion
    // 1 means suitable for all levels
    // 0 means no information is available, or level is not applicable
```

Reply:
I agree with the comment change, but why the level change? I think having 0 and -1 as special cases is much more common than 1, even if in this case, 1 can be used, as it's the default level. But overall, I think, 1 does not suggest a special case, so it's not worth using it here.

Reply:
I thought we had this discussion before in discord, and that we had resolved it..?

I really dislike using -1 as a special number for level. I think 0 much better fills the role of saying "level do not apply, or we don't know". You can never be level 0, so it is free to use for this purpose. The reason we're adding "1 means suitable for all levels" is not about demarcating 1 as a special number, but about helping users to understand how they should express "suitable for all".